.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# New Finality Provider
```

Comment:
Should this be the template for all new PRs?

Reply:
oh, no 
good catch, only for new fps

Reply:
I will rename the file

Reply:
Done at https://github.com/babylonchain/networks/pull/22/commits/0a869f34db4a763ba83a8a888e17885a0b784e44